.github/CODEOWNERS

```diff
@@ -120,6 +120,7 @@ pkg/pdatatest/                                           @open-telemetry/collect
 pkg/pdatautil/                                           @open-telemetry/collector-contrib-approvers @dmitryax
 pkg/resourcetotelemetry/                                 @open-telemetry/collector-contrib-approvers @mx-psi
 pkg/stanza/                                              @open-telemetry/collector-contrib-approvers @djaglowski
+pkg/translator/azure/                                    @open-telemetry/collector-contrib-approvers
```

Comment:
New modules must have an owner (different from the approvers)

Reply:
Is this technically a module? Also I don't think I can be a code owner or assign one.  I would nominate @atoulme if I had to (they originally wrote the code). But I think that's up to them.

Reply:
If the code already had an owner and it is just being moved, then I think it's fair to have the same owner as before, but it should be listed on .github/CODEOWNERS as such (otherwise our tooling won't work)

Reply:
Added as advised.